docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+
+MLFlow Wine classification
+===================
+
+Wine quality example
+(full code can be found `in GitHub repository <https://github.com/legion-platform/legion-mlflow/tree/master/examples/wine-quality>`_).
+
+In this tutorial you step by step will learn how to train, pack and deploy your model from scratch by using Legion Platform.
+
+Requirements
+~~~~~~~~~~~~~~~~~~~
+
+- You must have Legion Platform deployed in a cluster.
+- :term:`MLFlow <MLflow Model Training Toolchain Integration>` and :term:`docker <Docker REST API Packaging Toolchain Integration>` toolchain integrations must be installed.
+- ``legionctl`` is installed locally or Jupyter Lab with legion plugin is installed locally or in the cloud.
+
+Pre-sequences
+~~~~~~~~~~~~~~~~~~~
+
+To train, pack and deploy model you need to interact with Legion Platform EDI server. This server provide REST api.
+You can use it directly or using different tools.
+
+In this tutorial we will describe the process of interacting with EDI server using ``legionctl`` command line tool and
+Jupyter Lab with Legion Platform plugin.
+
+In the process of model training/packing/deployment Legion Platform should work with different kinds of data that you should provide.
+They are: source code of the train script, input data and so on.
+
+Legion Platform uses concept of ``Connections`` to manage all kinds of such data. Read more about ``Connections``.
+
+
+Authentication
+##################
+To interact with Legion Platform you should be authenticated via Keycloak. Read more about all available auth methods.
+
+`Using legionctl`
+
+Use interactive login mode for authenticate with Legion Platform
+Just type in terminal next command:
+
+.. code-block:: console
+
+    $ legionctl login --edi $EDI_URL
+
+Where ``$EDI_URL`` is host of edi server. Usually it has address ``https://edi.<legion-platform-host>``
+
+After running command above interactive session will be run and you will be invited to open the link that appeared in
+terminal.
+
+Click to the link and keycloak UI will be opened in your web browser. Two cases are possible:
+
+– You are already authenticated at keycloak. In this case you will see series of redirects and the result web page
+with suggestion to close this window already. Just return to terminal and you will see ``Success! Credentials have been saved.``
+
+– You have not been authenticated at keycloak yet. In this case you will see keycloak UI and should press ``Google`` button.
```

Comment:
I think we should not mention the Google authorization, Keycloak service, and an EPAM email because it's a configuration of our dev deployment.